commons/passive/garmin/garmin_generic_intensity.avsc

```diff
@@ -0,0 +1,43 @@
+{
+   "name":"GarminGenericIntensity",
+   "type":"record",
+   "namespace":"org.radarcns.passive.garmin",
+   "doc":"Intensity data from Garmin device.",
```

Comment:
Again, if possible, describe what the fields mean (what is high, moderate)

Reply:
Garmin devices compare your heartrate data to determine intensity level. It compares resting heartrate with current heartrate to determins Vigorous or Moderate intensity level. No specific ranges or algorithm specified anywhere. 

https://www.google.com/search?q=garmin+fenix+6+intensity+minutes&rlz=1C5CHFA_enIN763IN763&oq=garmin+fenix+6+intensity+&aqs=chrome.0.0j69i57j0.7869j0j4&sourceid=chrome&ie=UTF-8#kpvalbx=_eBPaXqfIHtmP4-EP3omL6Ak41

Reply:
updated in document. available in latest checkin.